src/test/java/edu/hawaii/its/api/service/TestMemberAttributeService.java

```diff
@@ -204,40 +139,81 @@ public void memberAttributesTest() {
     }
 
     @Test
-    public void membersAttributesTest() {
-        List<Subject> subjects = memberAttributeService.getMembersAttributes(ADMIN, testUids);
-        assertNotNull(subjects);
+    public void memberAttributeResultsTest() {
+        MemberAttributeResults results = memberAttributeService.getMemberAttributeResults(ADMIN, testUids);
+        assertNotNull(results);
+        HashSet<String> testUidsSet = new HashSet(testUids);
+        HashSet<String> testUhUuidsSet = new HashSet(testUhUuids);
+
+        for (MemberResult memberResult : results.getResults()) {
+            assertTrue(testUidsSet.contains(memberResult.getUid()));
+            assertTrue(testUhUuidsSet.contains(memberResult.getUhUuid()));
+        }
+
+        String testUid = testUids.get(0);
+        List<String> testList = new ArrayList<>();
+        testList.add(testUid);
+
+        // Should not contain memberResult of invalid uhIdentifier.
```

Comment:
To address Michael's comment, it would be easier to understand if you replaced "memberResult" with "member attributes".